### PR TITLE
Add date_wrapper_class option to the Date Form Type

### PR DESCRIPTION
**Pull request by NightFox7**
As discussed in the issue #1006, I added an option to the Date Form Type that I called "date_wrapper_class". This option takes an array with 3 elements "year", "month", "day". Each element would represent the corresponded input wrapper class.
The default value of "date_wrapper_class" is:
```
array(
    'year' => 'col-sm-4',
    'month' => 'col-sm-4',
    'day' => 'col-sm-4'
);
```

### CODE REVIEW

File: Form/Extension/DateTypeExtension.php [hunk 1]
```diff
@@ -22,7 +22,9 @@ public function finishView(FormView $view, FormInterface $form, array $options)
         if ('single_text' === $options['widget'] && isset($options['datepicker'])) {
             $view->vars['datepicker'] = $options['datepicker'];
         }
-
```
Comment (isometriks) [inline Form/Extension/DateTypeExtension.php]: Can you put this space back please?

Comment (NightFox7) [inline Form/Extension/DateTypeExtension.php]: May I ask why?

**Comment by isometriks** [comment]
Could I also convince you to write a test for this? You can copy from: https://github.com/phiamo/MopaBootstrapBundle/blob/master/Tests/Form/AddonLayoutTest.php and call it DateTypeLayoutTest. Just add 3 random class names and use xpath to make sure they are the same when rendered

**Comment by isometriks** [comment]
And do we think `sm` is a more sensible default than `xs`? I would think we'd pretty much always want them on one line

File: Form/Extension/DateTypeExtension.php (hunk 1 above)
Comment (isometriks) [inline Form/Extension/DateTypeExtension.php]: Should be a space between the two if statements. I think I might have commented on the wrong line

**Comment by isometriks** [comment]
Also I don't see where the config values are being passed into the extension? It should be setting a default, not checking if the option exists. If you don't provide those classes this would cause an error. You'll need to pass $options to the constructor and save them, then set the classes in setDefault

**Comment by NightFox7** [comment]
@isometriks Is this what you had in mind?

**Comment by isometriks** [comment]
Closer, you're missing the constructor in the extension now:

```php
public function __construct(array $options = array())
{
    $this->options = $options;
}
```

Make sure you define `$options` as well. You also have some extra spaces in the XML file as well. 8 spaces instead of 4.

**Comment by isometriks** [comment]
I think it looks pretty good now. Can you squash into one commit please? 
http://symfony.com/doc/current/contributing/code/patches.html#rebase-your-patch

**Comment by NightFox7** [comment]
I definitely did something wrong here! Sorry about that. Is there a way to fix this?

**Comment by NightFox7** [comment]
Why is the Travis CI build always failing?

**Comment by phiamo** [comment]
https://travis-ci.org/phiamo/MopaBootstrapBundle/jobs/47633452
looks like some things are not forward compatible

**Comment by NightFox7** [comment]
All 13 errors are related to the Test classes which I have not modified.

**Comment by isometriks** [comment]
Yeah, `setDefaultOptions` has now been deprecated. This is still safe to merge.

**Pull request merged**